core/Plugin/Manager.php

```diff
@@ -1191,6 +1191,8 @@ private function executePluginDeactivate($pluginName)
      */
     private function unloadPluginFromMemory($pluginName)
     {
+        $this->unloadPlugin($pluginName);
+
```

Comment:
Do you know why there is `unloadPluginFromMemory()` and `unloadPlugin()` and why one calls the other?